ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -11,6 +11,10 @@ base_images:
     name: assisted
     namespace: ocp-kni
     tag: assisted-installer-controller
+  assisted-operator-index-ocm-2.3:
+    name: ocm-2.3
+    namespace: openshift
+    tag: ocm-2.3
```

Comment:
The tag should be `assisted-service-index`

See docs for promotion stanza: https://docs.ci.openshift.org/docs/architecture/ci-operator/#publishing-to-an-integration-stream